guide/additional-info/miscellaneous-examples.md

```diff
@@ -150,7 +150,7 @@ message.channel.send({
 * You need at least two consecutive fields set to `inline` in order for them to display correctly.
 * If you have a thumbnail set, a maximum of 2 fields will be displayed inlined, as opposed to the default 3.
 * If you use the `.setTimestamp()` or `timestamp` field, the timestamp will automatically adjust the timezone depending on the user's device.
-* If your embed object variable is named `embed`, you can replace `.send({ embed: exampleEmbed })` with `.send({ embed })`
+<p class="tip"> If your embed object variable is named `embed`, you can replace <code>.send({ embed: exampleEmbed })</code> with <code>.send({ embed })</code></p>
```

Comment:
`<code>` tags aren't necessary for this; regular backticks should work (check the [guide/preparations/README.md file](https://github.com/Danktuary/Making-Bots-with-Discord.js/blame/master/guide/preparations/README.md#L49) for reference). There should also be a blank line between the `<p>` tag and the final list item.